packs/data/spell-effects.db/spell-effect-evolution-surge.json

```
@@ -0,0 +1,140 @@
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Sight",
+                        "value": "sight"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Scent",
+                        "value": "scent"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Amphibious",
+                        "value": "amphibious"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Speed",
+                        "value": "speed"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Large",
+                        "value": "large"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Climb",
+                        "value": "climb"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Huge",
+                        "value": "huge"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Fly",
+                        "value": "fly"
+                    }
```

```suggestion
                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Large",
                        "value": "large"
                    },
                    {
                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Climb",
                        "value": "climb"
                    },
                    {
                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Huge",
                        "value": "huge"
                    },
                    {
                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Fly",
                        "value": "fly"
                    }
```
These four should be predicated on the spell level, but since `item:level:x` isn't a roll option we can go by the actor level. `"predicate": [ { "gte": ["self:level", 5] } ]`, since this is a focus spell and your eidolon is always the same level as you we can get away with it.

---

Is there any way to make them not appear in the ChoiceSet prompt? my gut says no, but I'm not totally sure.

---

You can add a predicate to each Choice.